package.json

```diff
@@ -391,6 +391,11 @@
 					"type": "boolean",
 					"default": false,
 					"description": "Whether to automatically send a 'hot reload' request during a Flutter debug session when saving files."
+				},
+				"dart.analyzeAngularTemplates": {
```

Comment:
The comment in getAnalyserSettings confuses me somewhat. This setting change does not require an analyzer restart, but it does require a project load. The first half of the comment makes it seem like I should add the new setting, and the last half makes it seem like I shouldn't